deep-security-agent/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright:: 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+
+#######################
+# Mandatory Variables #
+#######################
+---
+# As you add agents to Deep Security, they are automatically made available for download through
+#   the same hostname:port as the Deep Security Manager admin interface. For Marketplace and software deployments, pass
+#   this data through the 'dsm_agent_download_hostname' & 'dsm_agent_download_port' attributes.
```

Comment:
Do you mean "dsa_agent_download_hostname" and "dsa_agent_download_port" here?

Reply:
Align all variables with Chef's.